File: 백준/Bronze/4892. 숫자 맞추기 게임/숫자 맞추기 게임.java
import java.util.*;
public class Main{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int cnt = 1;
        while(true){
            int n = sc.nextInt();
            if(n == 0)
                break;
            if(n%2 == 0){
                n = (((n*3)/2)*3)/9;
                System.out.println(cnt + ". even " + n);
            }
            else{
                n = ((((n*3)+1)/2)*3)/9;
                System.out.println(cnt + ". odd " + n);
            }
            cnt++;
        }
    }
}

File: 백준/Bronze/4892. 숫자 맞추기 게임/숫자 맞추기 게임.c
#include<stdio.h>
int main(void){
    int cnt = 1;
    while(1){
        int n;
        scanf("%d",&n);
        if(n == 0)
            break;
        if(n%2 == 0){
            n = (((n*3)/2)*3)/9;
            printf("%d. even %d\n",cnt,n);
        }
        else{
            n = ((((n*3)+1)/2)*3)/9;
            printf("%d. odd %d\n",cnt,n);
        }
        cnt++;
    }
        return 0;
    }